src/Components/DnDLayout/GridLayout.tsx

```diff
@@ -357,6 +358,10 @@ const GridLayout = ({ isLayoutLocked = false }: { isLayoutLocked?: boolean }) =>
           // eslint-disable-next-line @typescript-eslint/no-unused-vars
           activeLayout.map(({ widgetType, title, ...rest }, index) => {
             const { config } = getWidget(widgetMapping, widgetType);
+            const requiresOrgAdmin = isOrgAdminWidgetPermissionRequired(config);
+            if (requiresOrgAdmin && !currentUser?.is_org_admin) {
+              return null;
```

Comment:
I'm on the fence if we should remove the widget like I have here, or display some sort of placeholder indicating the user does not have permissions to view this widget. The latter would prevent oddness with the widgets moving around as well as the need for an immediate `PATCH` call to our update dashboard endpoint as the UI has auto-removed the widget.

I think this is more apparent if our base template includes an org admin widget, but we don't have separate base templates for org admins/non admins. This is also apparent if a user "copies" another dashboard in which they do not have access to all the widgets.

Reply:
For now, If you don't have permissions you should not be able to see it all. I think there will be a lot more nuance to this once we are done with the first iteration.